lbrynet/extras/daemon/Daemon.py

```diff
@@ -2598,8 +2598,8 @@ def jsonrpc_channel_list(self, account_id=None, page=None, page_size=None):
         elif old_txo.claim.is_signed and not clear_channel and not replace:
             channel = old_txo.channel
 
-        if 'fee_address' in kwargs:
-            self.valid_address_or_error(kwargs['fee_address'])
+        if 'fee_amount' in kwargs:
+            kwargs['fee_address'] = self.get_fee_address(kwargs, claim_address)
```

Comment:
mentioned this in slack as well but this introduces a bug where if you don't use `--replace` mode and just want to update the `fee_amount` this will overwrite the previously set `fee_address`